.travis.yml

```diff
@@ -0,0 +1,48 @@
+language: python
+dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
+python:
+    - 2.7
+    - 3.6
+    - 3.7
+env:
+    - TOXENV=django111
+    - TOXENV=django20
+    - TOXENV=django21
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=quality
+  exclude:
+    - python: 2.7
+      env: TOXENV=django20
+    - python: 2.7
+      env: TOXENV=django21
+  allow_failures:
+    - python: 3.6
+    - python: 3.7
+
+services:
+  - xvfb
+addons:
+    firefox: "52.0esr"
+install:
+    - "mkdir var"
+    - "wget https://github.com/mozilla/geckodriver/releases/download/v0.15.0/geckodriver-v0.15.0-linux64.tar.gz"
+    - "mkdir geckodriver"
+    - "tar -xzf geckodriver-v0.15.0-linux64.tar.gz -C geckodriver"
+    - "export PATH=$PATH:$PWD/geckodriver"
+    - "pip install -r requirements/dev.txt"
+    - "pip install -r requirements/test.txt"
+script:
+    - tox -e $TOXENV
+deploy:
```

Comment:
@cclauss deploying to pypi is beyond the scope of this INCR ticket. Could you remove this section?